core/src/main/java/com/linecorp/armeria/client/ClientOptions.java

```diff
@@ -71,8 +70,8 @@
 
     private static final ClientOptionValue<?>[] DEFAULT_OPTIONS = {
             DEFAULT_WRITE_TIMEOUT_MILLIS.newValue(DEFAULT_DEFAULT_WRITE_TIMEOUT_MILLIS),
-            DEFAULT_RESPONSE_TIMEOUT_MILLIS.newValue(DEFAULT_DEFAULT_RESPONSE_TIMEOUT_MILLIS),
-            DEFAULT_MAX_RESPONSE_LENGTH.newValue(DEFAULT_DEFAULT_MAX_RESPONSE_LENGTH),
+            DEFAULT_RESPONSE_TIMEOUT_MILLIS.newValue(Flags.defaultRequestResponseTimeoutMillis()),
+            DEFAULT_MAX_RESPONSE_LENGTH.newValue(Flags.defaultMaxRequestResponseLength()),
```

Comment:
What do you think about splitting these flags into four? defaultRequestTimeoutMillis, defaultResponseTimeoutMillis, defaultMaxRequestLength, defaultMaxResponseLength

Also, what about making defaultResponseTimeoutMillis is greater than defaultRequestTimeoutMillis so that a client has a higher chance of getting proper 503 Service Unavailable response when server-side timeout occurs? 15 sec?